### ISSUE TITLE
data dictionary pic

### ISSUE THREAD

**Issue by tntran** [tntran/ISQA-3420-Group-Project#7]
![image](https://cloud.githubusercontent.com/assets/16200170/11699209/d335fa48-9e88-11e5-8389-5ad445a64650.png)

**Comment by tntran** [comment]
![image](https://cloud.githubusercontent.com/assets/16200170/11699237/05c919fe-9e89-11e5-88d0-846535d1696f.png)

**Comment by tntran** [comment]
![image](https://cloud.githubusercontent.com/assets/16200170/11699250/13e5ca82-9e89-11e5-94ee-99f5e919403f.png)

**Comment by germonprez** [comment]
Hi guys,

I was checking out your repository and I'm not sure what you are doing. You
are simply supposed to be using the example from class (source code
intake). You are not supposed to be making your own system.

Matt

On Wed, Dec 9, 2015 at 3:23 PM, tntran notifications@github.com wrote:

> [image: image]
> https://cloud.githubusercontent.com/assets/16200170/11699209/d335fa48-9e88-11e5-8389-5ad445a64650.png
> 
> —
> Reply to this email directly or view it on GitHub
> https://github.com/tntran/ISQA-3420-Group-Project/issues/7.

## 

Mutual of Omaha Associate Professor
Information Systems
College of Information Science & Technology
University of Nebraska Omaha
http://ocrl.unomaha.edu/

**Comment by tntran** [comment]
we was under impression that we have to make our own rather then follow the slide.

Tien

**Comment by germonprez** [comment]
No. I had said this many times. This is why I set up times to meet with
your groups. You are simply supposed to be getting the material from class
up on GH. You naturally have to add some components on your own but the
context is supposed to be the intake of external source. I have this in the
details of the project as well:

In your group project, you are responsible for the design documentation in
the management of external source repositories.

matt

On Wed, Dec 9, 2015 at 3:33 PM, tntran notifications@github.com wrote:

> we was under impression that we have to make our own rather then follow
> the slide.
> 
> Tien
> 
> —
> Reply to this email directly or view it on GitHub
> https://github.com/tntran/ISQA-3420-Group-Project/issues/7#issuecomment-163399304
> .

## 

Mutual of Omaha Associate Professor
Information Systems
College of Information Science & Technology
University of Nebraska Omaha
http://ocrl.unomaha.edu/
